### DISCUSSION TITLE
I need Help with push notifications

### DISCUSSION THREAD

**Discussion by guillo270120** [AntlerVC/firetable#854] [Support / Q&A] [answered]
i want to know if there are some videotutorial to send push notifications with rowy

**Comment by BeeBombshell** [comment, marked as answer]
Hey @guillo270120. Currently, there's no such video tutorial for this. However, this can be achieved with the help of the Push Notification extension which utilizes the Firebase Cloud Messaging service. Here's a [link](https://github.com/rowyio/templates/pull/9/commits/16f658cc8b6805e1aede57fb942f64002ff7ff12) to a PR that documents the extension along with the template. If you have any specific questions regarding this, feel free to ask here or create a new discussion. Hope this helps!
